src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +39,36 @@
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{todo.id}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={() => setSelectedTodoId(0)}
```

Comment:
The method name `setSelectedTodoId` doesn't follow the naming conventions for methods. Consider renaming it to `handleSetSelectedTodoId` or `onSetSelectedTodoId`. Here is a resource that provides handy naming conventions for event handler functions/props in React: [Resource](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)